src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
```

Comment:
```suggestion
    <form
      action="#"
      data-qa="big"
      method="post"
      class="form-big"
```
Please style your elements using class selectors